src/main/java/DTOs/StudentConverter.java

```diff
@@ -1,10 +1,9 @@
-package springbootquickstarter.controllers;
+package DTOs;
```

Comment:
DTO should be under the app, not beside it. and the package name should not be upper case

Reply:
Done.